sirepo/pkcli/job_cmd.py

```diff
@@ -105,6 +105,14 @@ def _op(expect_file):
         ).does_api_reply_with_file(msg.api, msg.data.method)
         if x:
             with simulation_db.tmp_dir(chdir=True) as d:
+                if sirepo.feature_config.for_sim_type(template.SIM_TYPE).pdf_tmp_dir:
```

Comment:
Since this is generic let's make the name for the dir something more generic. Maybe file_reply_tmp_dir instead of pdf_tmp_dir.

In addition, I don't think this needs to be nested under simulation_db.tmp_dir. In this case we are using our own dir so we don't need the simulation_db.tmp_dir

Reply:
34d5c6d1b0aa5be7fb841e47a6fbf0fc34ecd9e9